### DISCUSSION TITLE
Error response from daemon

### DISCUSSION THREAD

**Discussion by rhw2008** [TransformerOptimus/SuperAGI#799] [Q&A]
Hello, 
I'm getting this error at the end of the build. I can't access localhost:3000 (This site can’t be reached.  localhost refused to connect.)

this is the error in visual studio: 

Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:3000 -> 0.0.0.0:0: listen tcp 0.0.0.0:3000: bind: An attempt was made to access a socket in a way forbidden by its access permissions.

thanks in advance

**Comment by rhw2008** [comment]
I had to kill all process and restart everything over.

**Comment by DMTarmey** [comment]
I had similar issue with new build i downloaded today, did you manage to resolve the issue .
